### PR TITLE
randomly shuffle team list in contributors page

### PR DESCRIPTION
**Pull request by aravind1078**
#795

### CODE REVIEW

File: website/static/js/index.js [hunk 1]
```diff
@@ -190,6 +204,8 @@ elements.overlay.addEventListener("touchstart", mobileToggleEvent, false);
 window.addEventListener("scroll", handleScroll, false);
 
 elements.colorSchemeSwitch.addEventListener("click", modeSwitch, false);
+const teamArr = Array.from(elements.teamList.getElementsByTagName("li"));
+randomShuffle(teamArr).forEach((li) => elements.teamList.appendChild(li));
```
Comment (sebmck) [inline website/static/js/index.js:208]:
```suggestion
for (const li of randomShuffles(teamArr)){
  elements.teamList.appendChild(li);
}
```

**Comment by sebmck** [comment]
Awesome, thank you! Looks great. I left a suggestion to use `for of` instead of `forEach` and committed it.

**Comment by aravind1078** [comment]
> Awesome, thank you! Looks great. I left a suggestion to use `for of` instead of `forEach` and committed it.

@sebmck should i do those changes or its fine ?

**Comment by sebmck** [comment]
@aravind1078 Nah you don't need to. I already accepted the suggestion above.

**Pull request merged**